src/app/team/documents/proofs/Edit.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component, createRef } from 'react';
+
+import Header from '../../../../components/header';
+import Button from '../../../../components/button';
+import Selector from '../../../../components/selector';
+import Input from '../../../../components/input';
+
+const DEFAULT_PROOFS = [
+  'None',
+  'Photo ID',
+  'Proof of address',
+  'Proof of income',
+  'Birth certificates',
+];
+
+class ProofsRequiredEdit extends Component {
+  constructor(props) {
+    super(props);
+
+    let proofs = [];
+
+    if (this.props.value) {
+      proofs = this.props.value.map(el => el.document);
+    }
+
+    this.state = {
+      proofs,
+      showForm: false,
+    };
+
+    this.customProofRef = createRef();
+    this.updateValue = this.updateValue.bind(this);
+    this.showForm = this.showForm.bind(this);
+    this.onSubmitProofForm = this.onSubmitProofForm.bind(this);
+    this.onProofClick = this.onProofClick.bind(this);
+  }
+
+  onSubmitProofForm(e) {
+    e.preventDefault();
+
+    const newValue = this.customProofRef.current.element.value;
+
+    this.setState(prevState => ({
+      proofs: [
+        ...prevState.proofs,
+        newValue,
+      ],
+      showForm: false,
+    }));
+  }
+
+  onProofClick(name) {
+    let { proofs } = this.state;
+
+    if (name === 'None') {
+      proofs = ['None'];
```

Comment:
I must have missed it in the review, but do we actually end up sending "None" to the backend, and storing it in the DB, or is it filtered out at some point?
If so, I think that's a good idea (so we can distinguish unknown proofs from known-to-be-empty proofs), but given our current user-facing code doesn't know about this, we might end up showing the user something like "Requires None".

So, if that is the case, please just let me know, so I can make a ticket for updating the user-facing page accordingly.